maps/tests/script.js

```diff
@@ -1,7 +1,37 @@
 console.log('SCRIPT LAUNCHED');
-WA.sendChatMessage('Hi, my name is Poly and I repeat what you say!', 'Poly Parrot');
+//WA.sendChatMessage('Hi, my name is Poly and I repeat what you say!', 'Poly Parrot');
```

Comment:
The "script.js" file is an actual test script for test scenarios.
It would be better if you used another file for your tutorial map.